storybookjs/storybook
# Dependency: bump `markdown-to-jsx` to v7.4.5

Don't see any breaking change in change logs of the `markdown-to-jsx`. 
This version can resolve https://github.com/quantizor/markdown-to-jsx/issues/529 and other bugs







## What I did

- bump `markdown-to-jsx` to `v7.4.5` in workspace `@storybook/blocks`
- bump `markdown-to-jsx` to `v7.4.5` in workspace `@storybook/manager`

```sh
yarn workspace @storybook/blocks add markdown-to-jsx
yarn workspace @storybook/manager add markdown-to-jsx
```



## Checklist for Contributors

### Testing



#### The changes in this PR are covered in the following automated tests:

#### Manual testing

I thing the follow testing is required. 
And should be add to auto test

```md
// test.md

[![latest version](https://img.shields.io/npm/v/markdown-to-jsx.svg?label=latest%20%20version)](https://www.npmjs.org/package/markdown-to-jsx)

#
##

----

other basic feature of Markdown
```

```mdx
// @storybook/block -> Markdown.test.stories
import { Markdown } from "@storybook/blocks";
import Readme from "./test.md?raw";

<Markdown>{Readme}</Markdown>
```



### Documentation



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

## Checklist for Maintainers

   <details>
     <summary>Available labels</summary>

     - `bug`: Internal changes that fixes incorrect behavior.
     - `maintenance`: User-facing maintenance tasks.
     - `dependencies`: Upgrading (sometimes downgrading) dependencies.
     - `build`: Internal-facing build tooling & test updates. Will not show up in release changelog.
     - `cleanup`: Minor cleanup style change. Will not show up in release changelog.
     - `documentation`: Documentation **only** changes. Will not show up in release changelog.
     - `feature request`: Introducing a new feature.
     - `BREAKING CHANGE`: Changes that break compatibility in some way with current major version.
     - `other`: Changes that don't fit in the above categories.
   
   </details>

### 🦋 Canary release



This PR does not have a canary release associated. You can request a canary release of this pull request by mentioning the `@storybookjs/core` team here.

_core team members can create a canary release [here](https://github.com/storybookjs/storybook/actions/workflows/canary-release-pr.yml) or locally with `gh workflow run --repo storybookjs/storybook canary-release-pr.yml --field pr=<PR_NUMBER>`_
